nixos/modules/services/mail/dovecot.nix

```diff
@@ -297,10 +298,15 @@ in
 
 
   config = mkIf cfg.enable {
-
     security.pam.services.dovecot2 = mkIf cfg.enablePAM {};
 
-    services.dovecot2.protocols =
+    security.dhparams = mkIf (! isNull(cfg.sslServerCert)) {
```

Comment:
Just a presentation note, wouldn't the usual style in nixpkgs be `! isNull cfg.sslServerCert` here?

Reply:
Oh indeed, I was mostly thinking about the parenthesis-around-a-single-argument point :)